lib/logger/lib/logger/backends/console.ex

```diff
@@ -3,25 +3,29 @@ defmodule Logger.Backends.Console do
 
   use GenEvent
 
-  def init(_) do
-    if user = Process.whereis(:user) do
-      Process.group_leader(self(), user)
-      {:ok, configure([])}
+  def init(:console) do
+    if Process.whereis(:user) do
+      init({:user, []})
     else
       {:error, :ignore}
     end
   end
 
-  def handle_call({:configure, options}, _state) do
-    {:ok, :ok, configure(options)}
+  def init({device, opts}) do
+    {:ok, configure(device, opts)}
   end
 
-  def handle_event({_level, gl, _event}, state) when node(gl) != node() do
+  def handle_call({:configure, options}, state) do
```

Comment:
Right, `put_env` is only happening for `:console`: https://github.com/elixir-lang/elixir/pull/3390/files#diff-1fc817aa6e70de5c0af295c16d35280eR47.